mxcube3/routes/qutils.py

```diff
@@ -822,18 +844,18 @@ def _queue_add_item_rec(item_list, current_queue):
     Each item (dictionary) describes either a sample or a task.
     """
     children = []
-
     for item in item_list:
         item_t = item["type"]
         # If the item a sample, then add it and its tasks. If its not, get the
         # node id for the sample of the new task and append it to the sample
         sample_id = str(item["sampleID"])
 
         # Do not add samples that are already in the queue
-        if item_t == "Sample" and item["sampleID"] not in current_queue:
-            sample_node_id = add_sample(sample_id, item)
+	# TODO: fix this!!!
```

Comment:
In this case I would sugest:

 if item["sampleID"] not in current_queue and item_t == "Sample": 

What is the TODO: fis this about ?

Reply:
A temporary fix for the addition of samples, I had troubles with this before. I know you commit some fixes and perhaps this is no longer needed, but I need to check here.